docs/rocketchat/get-help-in-rocketchat.md

```diff
@@ -1,95 +1,99 @@
+    **Please note:** Screenshots, while helpful, won’t be accessible without alt text.  
 
-**If the problem is with a tool**
+2. After detailing the problem, we want to know what troubleshooting you tried. Being proactive helps us narrow down solutions and stops repetition
 
-What tool are you trying to use? Let the community know which platform tool is causing the problem.
+    Please give us a link to the source if you followed steps on a StackOverflow thread or read documentation about the problem
 
-**If the problem is with a stack**
+3. Last, please tell us what type of help you’re looking for. A solution, link to resources, and/or another way to complete a task? Please be very clear so we don’t resort to guessing
 
-Tell us about your stack. What languages and libraries are you using, and which versions of these?
+## The 'where'
 
-**Additional information**
+Tell us about the environment the problem happened in: 
+-	In a cluster, let us know the namespace and cluster name
+- With a stack, let us know what languages and libraries you used, including which versions
+- With a tool, please specify which platform tool
+- With a web application, please specify which browser, browser version, and device
+     - A Mac, a Windows machine, an iPhone, an Android, or another device?
 
-Also explain what device you were using when you experienced the problem. Were you using a Windows machine, a Mac, an iPhone, an Android or something else? If you were using a web application, what browser were you using? Provide as much detail as possible, including version numbers.
+ ## The 'when'
 
----
-## What are you trying to do?
-
- Explain what task you were trying to accomplish.
- 
--  How you were doing it, and why it was important. The more detail you can provide the better. 
-- Let the community know what you were trying to do and what outcome you had expected.
-
----
-## What is the problem?
+ Timestamps help! Please consider letting us know: 
+- The date
+- The time, including time zone if different from P.S.T. 
+- If working on a legacy task, the timestamp of the original work
 
-Now that the community knows what was supposed to happen, tell them what went wrong. Include descriptions of
 
-- How the problem occurred
-- What the problem looks like and how it’s behaving. For example, if a screen is failing to load properly, explain how it looks and why it’s not correct. 
-- Provide as much detail as possible. If you received an error message, include a screenshot of the message with your question. 
+## Post the question
 
-##  What have you done to fix the problem?
+When you finish writing the detailed question, please revisit [Jon Skeet's checklist](https://codeblog.jonskeet.uk/2012/11/24/stack-overflow-question-checklist/) to make sure you're being efficient. Then, post it in the best Rocket.Chat channel. 
```

Comment:
- should provide some help/link here for "best Rocket.Chat channel"

Reply:
> * should provide some help/link here for "best Rocket.Chat channel"

added link to channel descriptions